backend/src/main/java/gov/cdc/usds/simplereport/db/model/PatientLink.java

```diff
@@ -3,14 +3,15 @@
 import java.time.Duration;
 import java.time.Instant;
 import java.util.Date;
+import java.util.UUID;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.JoinColumn;
-import javax.persistence.OneToOne;
+import javax.persistence.ManyToOne;
 
 @Entity
 public class PatientLink extends EternalAuditedEntity {
-  @OneToOne(optional = false)
```

Comment:
This mapping was incorrect to the underlying data model – it was also preventing me from correctly back-referencing TestOrders to PatientLinks in the data loader

Reply:
Good catch